code/src/containers/category-section/components/CategoryItem.tsx

```diff
@@ -1,22 +1,26 @@
 import AppBox from "@/components/app-box/AppBox";
+import AppLink from "@/components/app-link/AppLink";
 import AppTypography from "@/components/app-typography/AppTypography";
 
 type CategoryItemProps = {
   label: string;
   image: string;
+  href: string;
 };
 
-const CategoryItem = ({ label, image }: CategoryItemProps) => {
+const CategoryItem = ({ label, image, href }: CategoryItemProps) => {
   return (
     <AppBox
```

Comment:
I decided to make only the image clickable instead of the item as a whole, to help the user avoid clicking on the white space next to the category image and selecting a different category by mistake.